### PR TITLE
FM2-512: FHIR IG Observation Resource Example

### PR DESCRIPTION
**Pull request by abertnamanya**
Issue Ticket: https://issues.openmrs.org/browse/FM2-512

Screenshot from 2022-09-09 23-56-51

Screenshot from 2022-09-09 23-56-37

### CODE REVIEW

**Review by pmanko — APPROVED** [review]

File: input/fsh/omrs-observation.fsh [hunk 1]
```diff
@@ -15,19 +15,34 @@ Description: "Observation resource created by a OpenMRS that representing a resu
...
 
-// OpenMRS Task  Example
+/* OpenMRS Observation Resource  Example */
+
 Instance: example-openmrs-Observation
 InstanceOf: OpenMRSObservation
 Usage: #example
 Title: "Openmrs Observation"
 Description: "Example OMRS Observation Resource"
 * identifier.value = "e8f8d71a-6c14-4105-b6fe-30f3cf6b7dc9"
```
Comment (pmanko) [inline input/fsh/omrs-observation.fsh:25]: I don't think we map an identifier field in the translator: https://github.com/openmrs/openmrs-module-fhir2/blob/master/api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ObservationTranslatorImpl.java

**Review by pmanko — CHANGES_REQUESTED** [review]
double-check that all defined fields are being mapped in the translator: https://github.com/openmrs/openmrs-module-fhir2/blob/master/api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ObservationTranslatorImpl.java

**Review by pmanko — APPROVED** [review]

**Comment by abertnamanya** [comment]
@pmanko  I think I have messed up the code by trying to resolve the conflict in the PR, even other commits of different branches are coming in.

**Comment by abertnamanya** [comment]
@pmanko How can I resolved the issue of combining multiple commits

**Pull request merged**